LayoutTests/fast/css/bdi-element-invalid-dir.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/dom.html#the-directionality">
+<style>
+div { position: relative; width: 100px; height: 100px; background: red; }
+bdi { width: 100px; height: 100px; display: block; }
+span { display: inline-block; width: 50px; height: 100px; background: green; color: green; }
+#red { background: green; position: absolute; left: 0px; top: 0px; }
+</style>
+</head>
+<body>
+<div><bdi dir="foo"><span id="red"></span><span>&#x05EA;</span></bdi></div>
```

Comment:
It's confusing that the `span` is `id="red"`, since it isn't red itself. (It's covering the left half of the red `div`.) Maybe this could be `id="left"`?

Reply:
Yeah, the meaning of this span changed when I fixed iOS simulator failure. Will rename it to left.